### PR TITLE
CLDR-14423 Use year marker " 'р'." consistently in uk

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-14423 - Use the year marker 'р'." consistently in Ukrainian

Please check spreadsheet attached to issue for additional feedback from the linguist

- [x] This PR completes the ticket.

### CODE REVIEW

**Comment by AEApple** [comment]
@pedberg-icu - Can you please take a look and let me know if I missed any items I should update

- I didn't add the proposed year marker for Gy, which the language specialist suggested please let me know if that does align with your expectation it wasn't clear in the issue.
- The language specialist flagged a couple of LLL forms that they believe should be MMM instead, can you let me know what you think? Happy to push that part of the feedback to 41 in another ticket if needed.
- In the interval formats I went with "'рр'." at the end if nothing followed the y but "'р'." if G followed the year for example, but let me know if your feedback agrees with our language specialist that "'р'." should always follow the y instead of using "'рр'." when the year is part of the interval on both sides.
- Our language specialist also suggested that "'р'." should be in the interval formats with MM as well as MMM. I didn't make that change since it does not align with your reported feedback. I will probably include in the clone ticket for 41 unless you think it should be implemented now.

**Comment by srl295** [comment]
@AEApple I don't see a spreadsheet?

**Review by srl295 — COMMENTED** [review]
data seems OK but i don't have any background to compare it to.

**Comment by AEApple** [comment]
> @AEApple I don't see a spreadsheet?

It's included in the Jira issue, I actually exported as a PDF, let me know if it doesn't work for you and I can reattach. (Actual spreadsheet is linked from the TC notes.)

**Comment by srl295** [comment]
> > @AEApple I don't see a spreadsheet?
> 
> It's included in the Jira issue, I actually exported as a PDF, let me know if it doesn't work for you and I can reattach. (Actual spreadsheet is linked from the TC notes.)

Attached to https://unicode-org.atlassian.net/browse/CLDR-14423 ? I don't see anything attached there.

**Comment by AEApple** [comment]
Weird my update didn't publish, so I just published the saved draft.

**Review by srl295 — APPROVED** [review]
spot check lgtm

File: common/main/uk.xml [hunk 1]
```diff
@@ -1592,9 +1592,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 						<dateFormatItem id="yyyyMd">dd.MM.y G</dateFormatItem>
 						<dateFormatItem id="yyyyMEd">E, dd.MM.y G</dateFormatItem>
 						<dateFormatItem id="yyyyMMM">LLL y G</dateFormatItem>
```
Comment (stenshamn) [inline common/main/uk.xml:1594]: The ticket says "Input from our sources suggests that 'р' should be used for all formats with MMM/LLL or longer month styles." so then we should be changing this line as well, no?

**Review by stenshamn — APPROVED** [review]

**Pull request merged**